src/cores/events/EventsList.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { Link } from 'react-router-dom'
+import { addEvent, resetEvents, setAmount } from '../../features/events/eventSlice'
+import * as eventsServices from '../../services/events.services'
+import Event from './Event'
+import Pagination from './Pagination'
+
+function EventsList() {
+    const postPerPage = 9;
+    const page = useSelector(state => state.events.page);
+    const events = useSelector(state => state.events.events);
+    const amount = useSelector(state => state.events.amount)
```

Comment:
```suggestion
    const { page, events, amount } = useSelector(({ events: { page, events, amount } }) => ({ page, events, amount }));
```

Reply:
O quizas hasta mejor...
```suggestion
    const { page, events, amount } = useSelector(state => state.events);
```